x-pack/platform/plugins/shared/cases/common/types/domain/suggestion/v1.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { z } from '@kbn/zod';
+import type { CaseAttachmentWithoutOwner } from '../attachment/v1';
+
+const suggestionOwnerSchema = z.enum(['observability', 'security', 'stack']);
```

Comment:
I would have these names match the current owners format from `x-pack/platform/plugins/shared/cases/common/constants/owners.ts`